src/components/Interactive.js

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash/fp'
+import { forwardRef } from 'react'
+import { h } from 'react-hyperscript-helpers'
+
+
+const allowedHoverVariables = ['backgroundColor', 'color', 'boxShadow', 'opacity', 'textDecoration']
+const pointerTags = ['button', 'area', 'a', 'select']
+const pointerTypes = ['radio', 'checkbox', 'submit', 'button']
+
+const Interactive = forwardRef(({
+  className = '', as, type, role, onClick, onKeyDown, disabled, children, tabIndex, hover = {}, style = {}, ...props
+}, ref) => {
+  const { cursor } = style
+
+  const onClickPointer = !disabled && (onClick ||
+    pointerTags.includes(as) ||
+    pointerTypes.includes(type)) ? { cursor: 'pointer' } : {}
+  const computedCursor = cursor ? { cursor } : onClickPointer
+
+  const onClickTabIndex = onClick ? 0 : undefined
+  const computedTabIndex = _.isNumber(tabIndex) ? tabIndex : onClickTabIndex
```

Comment:
(from mob) This is allowing tabbing to disabled elements, we should figure out if we want to allow that (current behavior you cannot). Worth considering allowing tabbing to disabled element so keyboard users can see tooltip.

Reply:
After reviewing some a11y guidelines I kept the behavior as it currently is in Terra. The guidelines recommend excluding disabled elements from the tab order. Also, when trying this out on the workflow page, where there can be many disabled nodes keeping those items in the tab order seemed detrimental to the UX. However, we should take care when adding tooltips to disabled nodes as those tooltip messages will not be accessible to anyone not using a pointing device.